lib/nexpose/scan.rb

```diff
@@ -407,9 +407,10 @@ def import_scan(site_id, zip_file)
       data = Rex::MIME::Message.new
       data.add_part(site_id.to_s, nil, nil, 'form-data; name="siteid"')
       data.add_part(session_id, nil, nil, 'form-data; name="nexposeCCSessionID"')
-      scan = ::File.new(zip_file, 'rb')
-      data.add_part(scan.read, 'application/zip', 'binary',
-                    "form-data; name=\"scan\"; filename=\"#{zip_file}\"")
+      ::File.new(zip_file, 'rb') do |scan|
```

Comment:
> I don't think so in this case, to my knowledge usually we use :: to call Module::Class to know which class we are calling. Could you please remove the ::?

Using ::File over File I assume at one point (if not still the case) we had a conflicting constant in the Nexpose module namespace. Using ::File explicitly gets the top level constant (out of the Nexpose namespace) and leaving it alone doesn't affect this fix (while changing it could introduce a bug theoretically).